ui/pages/confirmations/hooks/useConfirmationNavigation.ts

```diff
@@ -69,11 +69,18 @@ export function useConfirmationNavigation() {
     [confirmations, navigateToId],
   );
 
-  const navigateNext = useCallback(() => {
-    if (count > 1) {
-      navigateToIndex(1);
-    }
-  }, [count, navigateToIndex]);
+  const navigateNext = useCallback(
+    (confirmationId: string) => {
+      const pendingConfirmations = confirmations.filter(
```

Comment:
How is this different to just checking `count`, assuming one of the confirmations is always the current one?